docs/getting_started.md

```diff
@@ -0,0 +1,80 @@
+# Getting Started
+
+Cortex can be runs as a single binary or as multiple independent microservices.
+
+For simplicity & to get started, we'll run it as a single binary:
+
+```sh
+$ make protos # Build all the protobuffer definitions.
+$ go build ./cmd/cortex
+$ ./cortex -config.file=./cmd/cortex/single-process-config.yaml
+```
+
+This starts a single Cortex node storing chunks and index to your local filesystem in `/tmp/cortex`.
+It is not intended for production use.
+
+Add the following to your prometheus config:
+
+```yaml
+remote_write:
+- url: http://localhost:9009/api/prom/push
+```
+
+And start Prometheus with that config file:
+
+```sh
+$ ./prometheus --config.file=./documentation/examples/prometheus.yml
+```
+
+Your Prometheus instance will now start pushing data to Cortex.  To query that data, start a Grafana instance:
+
+```sh
+$ docker run -p 3000:3000 grafana/grafana
+```
+
+In [the Grafana UI](http://localhost:3000) (username/password admin/admin), add a Prometheus datasource for Cortex (`http://host.docker.internal:9009/api/prom`).
+
+## Scale out
+
```

Comment:
Maybe a little section about prerequisites to scale out cortex? It seems Cortex needs Consul for the cluster coordination. Does it need something else?

Reply:
I've made the following sentence a list of requirements.